about.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
```

Comment:
- [ ] The same thing for the `about `page, kindly use the given `colors(\#272A31, #EC5242, #D3D3D3)` and `fonts(Cocogoose, Lato)` for every section.